### DISCUSSION TITLE
Newer version of moose fails to execute on more than a specific number of nodes on a HPC cluster

### DISCUSSION THREAD

**Discussion by morsi95** [idaholab/moose#26900] [Q&A General]
Hello!

I use a moose based app for my simulations on a linux cluster. Usually on older moose releases, I was capable of running my simulations on 24 nodes using MPI tasks and threads (e.g SLURM setup : --nodes=24 --cpus-per-task=7 --ntasks-per-node=4). But now I have to use the newer moose release (post Oct 2022 releases) and now I can't use more than 8 nodes. I have a mesh of around 8 million elements and my executable link looks like the following: mpiexec   -n 96 mooseapp-opt --distributed-mesh -i  INPUT.i  --n-threads=7
Keep in mind that the mesh did not change at all and no errors are received when running tests. Also, the main reason why I used to use threading before is because it allowed me to use many nodes in moose older releases. Now however, it doesn't matter if I use threads or not because in both ways I will only be able to run on up to 8 nodes only. Could someone help me, please?

**Comment by GiudGiud** [comment]
Hello

Can you please paste the actual error you are hitting? MOOSE should still work with any number of nodes

Guillaume

**Reply by morsi95** [reply]
Unfortunately, the log file shows no error when the job is running. It will just not be executed and will stay blank until the time limit is reached.

**Reply by GiudGiud** [reply]
Did you rule out a problem with your job scheduler?
Did you try running a simple command instead of moose

Is there maybe a `.err` file created which has the errors?

**Reply by morsi95** [reply]
Yes, I ran a simple command and it seems it is not a problem with the scheduler or with mpi. 
There are no .err file available but the output log returns nothing with the running moose job.
The input files, including the job submission file, are all the same before and after updating moose. The only thing that changed was the moose release.

**Reply by GiudGiud** [reply]
and moose works with less processes? you still have successful runs with 4  nodes?

**Reply by morsi95** [reply]
Yes. It works with 8 or less nodes.

**Reply by GiudGiud** [reply]
Without a log we can only guess what s happening. 
I'd say try a different timpi synch algorthim (use --help on the app to see what I m talking about)
then rebuild with a different MPI if that doesnt work
